src/ott/tools/soft_sort.py

```diff
@@ -499,31 +500,33 @@ def multivariate_cdf_quantile_maps(
       to be uniform by default.
     kwargs: keyword arguments passed on to the :func:`~ott.solvers.linear.solve`
       function, which solves the OT problem between ``inputs`` and ``targets``
-      using the Sinkhorn algorithm.
+      using the :class:`~ott.solvers.linear.sinkhorn.Sinkhorn` algorithm.
```

Comment:
here i wasn't sure about the reference, because we use `solve`... that being said, probably not a good idea to use `LR` on this, because would crash :) so maybe an instance where we should force kwargs to only refer to sinkhorn...